remotespark/kernels/kernelmagics.py

```diff
@@ -190,7 +195,7 @@ def cleanup(self, line, cell="", local_ns=None):
             self.ipython_display.send_error("When you clean up the endpoint, all sessions will be lost, including the "
```

Comment:
This is not related to this PR. It's better to set a string variable with this sentence then print it.